components/post_view/floating_timestamp/floating_timestamp.tsx

```diff
@@ -12,17 +12,16 @@ const DATE_RANGES = [
 ];
 type Props = {
     isScrolling: boolean;
-    isMobile: boolean;
     createAt: Date | number;
     toastPresent: boolean;
     isRhsPost: boolean;
 }
 
 export default class FloatingTimestamp extends React.PureComponent<Props> {
     render() {
-        const {isMobile, createAt, isScrolling, isRhsPost, toastPresent} = this.props;
+        const {createAt, isScrolling, isRhsPost, toastPresent} = this.props;
 
-        if (!isMobile || createAt === 0) {
+        if (createAt === 0) {
```

Comment:
This seems like a UX change here. What's the effect of this change?

Reply:
It's not actually a change. The `FloatingTimestamp` is only actually mounted in mobile view (https://github.com/mattermost/mattermost-webapp/pull/9508/files#diff-ec9be19b2c781da350707675e6fe6a8483f55db7deaee8737edb55cc8ba79637R574), so this check is redundant